modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -133,10 +136,14 @@ public void testBuildTemplate() throws Exception {
 			"mvc-portlet", "hello-world-portlet", "com.test",
 			"-DclassName=HelloWorld", "-Dpackage=hello.world.portlet");
 
+		Path gradleOutputPath = new File(
```

Comment:
Lets not specify the gradleOutputPath and mavenOutputPath for each test.  They are always the same.

Just simplify the `_buildProjects` to take a `gradleProjectDir` and  `mavenProjectDir` and then down in that method we can calculate the default maven/gradle output path as `build/libs` and `target`